### DISCUSSION TITLE
SIP 1 - DKG

### DISCUSSION THREAD

**Discussion by alonmuroch** [ssvlabs/SIPs#7] [SIP Discussions]
This is a discussion board for SIP 1

Open issues:

- [ ] dos protection

**Comment by alonmuroch** [comment]
@amir-blox @lior-blox one thing I'm missing is DDOS protection.. basically what prevents anyone from creating a million DKG requests and flooding the network.
Ideas?

**Reply by amirylm** [reply]
As pubsub is used for messaging, we will use scoring as protection against spammers (that flood the network with invalid requests).
In addition, we should apply rate limiting to ensure no one can overload the topic, even with valid requests.

**Reply by alonmuroch** [reply]
can you suggest how we can do it? It seems important to ad to this SIP

**Comment by GalRogozinski** [comment]
DKG is out of protocol now
